src/main/java/frc/robot/subsystems/RollerSubsystem.java

```diff
@@ -84,6 +89,19 @@ public void openMotor() {
         openTimer.start();
     }
 
+    public InstantCommand getOuttakeCommand(double power, double duration){
+        return(new InstantCommand(() -> {
+            roll(power, duration);
+        }));
+    }
+
+    public void roll(double power, double duration){
```

Comment:
If the above command is the only place where this method is used, it can be inlined. Also, why is constructing the command (and timer / states) within the subsystem necessary?